cf/src/connection.js

```diff
@@ -1,9 +1,5 @@
-import { process } from '../polyfills.js'
 import { Buffer } from 'node:buffer'
-import { setImmediate, clearImmediate } from '../polyfills.js'
-import { net } from '../polyfills.js'
-import { tls } from '../polyfills.js'
-import { crypto } from '../polyfills.js'
+import { setImmediate, clearImmediate, process, net, tls, crypto } from '../polyfills.js'
```

Comment:
Reworked `polyfill.cf.js` to allow for a single `../polyfills.js` import (eslint was not happy about the multiple imports)